helpers/i18n.js

```diff
@@ -2,21 +2,26 @@ const i18next = require('i18next');
 const Backend = require('i18next-sync-fs-backend');
 const fs = require('fs');
 const path = require('path');
+const { Configuration } = require('@schul-cloud/commons');
 const logger = require('./logger');
 
-const defaultLanguage = 'de';
-const localeDir = path.join(__dirname, '../locales');
+const i18nDebug = Configuration.get('I18N__DEBUG');
+const i18nFallbackLanguage = Configuration.get('I18N__FALLBACK_LANGUAGE');
+const i18nDefaultLanguage = Configuration.get('I18N__DEFAULT_LANGUAGE');
 
+const localeDir = path.join(__dirname, '../locales');
```

Comment:
please add caching layer for the available language

Reply:
Moved auto-detection to config